root/etc/s6-overlay/s6-rc.d/init-crontabs-config/run

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/with-contenv bash
```

Comment:
According to https://github.com/just-containers/s6-overlay this is the old way of calling these scripts.

Reply:
We are aware, it's intentional.

Reply:
Explained on discord, but in case anyone else comes looking:
https://github.com/linuxserver/docker-baseimage-alpine/blob/master/root/usr/bin/with-contenv is included in our base images so that we did not have to adjust the path in every one of our downstream images when we migrated from s6v2 to s6v3. We later also added the umask logic as well.